src/tezos/TezosMessageUtil.ts

```diff
@@ -0,0 +1,110 @@
+/*eslint no-bitwise: 0*/
+import base58check from "bs58check";
+import baseN from "base-n";
+
+const base128 = baseN.create({
+  characters: [...Array(128).keys()].map(k => ("0" + k.toString(16)).slice(-2))
+});
+
+export namespace TezosMessageUtils {
```

Comment:
Could we leave a comment about which kinds of functions this namespace contains?